src/Bugsnag/Client.php

```diff
@@ -292,7 +308,7 @@ public function setType($type)
      */
     public function setMetaData(array $metaData)
     {
-        $this->config->metaData = $metaData;
+        $this->config->metaData = array_merge_recursive($this->config->metaData, $metaData);
```

Comment:
We probably shouldn't change this, because now we're unable to remove existing entries. Instead, we should add a new method, perhaps `addMetaData`?